server-spi-private/src/main/java/org/keycloak/models/utils/DefaultRequiredActions.java

```diff
@@ -83,6 +83,17 @@ public static void addActions(RealmModel realm) {
             realm.addRequiredActionProvider(termsAndConditions);
         }
 
+        if (realm.getRequiredActionProviderByAlias(UserModel.RequiredAction.GRANT_MANAGEMENT.name()) == null) {
```

Comment:
Considering that Grant Management might be preview feature, it might be good if it is not included as default and keycloak admin explicitly add corresponding required action afterwards. WDYT?

Reply:
true, I will remove.